packages/desktop-gui/src/lib/utils.js

```diff
@@ -1,6 +1,31 @@
 import _ from 'lodash'
 import gravatar from 'gravatar'
-import duration from '../../../server/lib/util/duration'
```

Comment:
yah, it would be nice to sync this back with the server after it's updated to remove moment.js also